pkg/k8s/annotate.go

```diff
@@ -31,6 +32,11 @@ type K8sClient struct {
 	kubernetes.Interface
 }
 
+// K8sSlimClient is a wrapper around slimclientset.Clientset.
+type K8sSlimClient struct {
+	slimclientset.Interface
+}
+
```

Comment:
Out of curiosity, why do we need this wrapper?

Reply:
I don't know why all the clients are wrapped like this in structs in the k8s package. Should I remove them?